### DISCUSSION TITLE
Scrolling down only one display line when text line is wrapped over multiple lines

### DISCUSSION THREAD

**Discussion by opsidjflksdf** [microsoft/vscode-discussions#1183] [Extension Development QnA]
When there's a long line of file text that spreads over multiple (display) lines in the text editor (with Word Wrap being enabled), if the lines in question are at the very top of the document, and we press Ctrl-Down to scroll the document one line down, it scrolls the whole 3-4 display lines that make up the entire file line. I'm looking for the functionality where it would only scroll down one display line.
That's the behavior we get (only one display line from the top being scrolled) if for example instead of using Ctrl-Down, we place the cursor on the last line and press Down - it will scroll the whole document by one display line - so the lines at the top will move out of view one by one, rather than all at once.
I was looking at the extensions API and saw TextEditor.RevealRange - that seems to work with file lines only. Would there be a way to use it for display lines? 
Thanks in advance for any help with this.

**Comment by isidorn** [comment]
What is your extension trying to achieve exactly? What is your extension? Are you still facing this issue? Thanks

**Comment by olguzzar** [comment]
Closing this discussion as it's been a while without any updates - please feel free to reopen if you have new information!
